pkg/trustedresources/verify_test.go

```diff
@@ -350,9 +377,13 @@ func TestVerifyTask_Error(t *testing.T) {
 	for _, tc := range tcs {
 		t.Run(tc.name, func(t *testing.T) {
 			err := VerifyTask(ctx, tc.task, k8sclient, tc.source, tc.verificationPolicy)
-			if !errors.Is(err, tc.expectedError) {
+			verificationErr := err.(*VerificationError)
+			if !errors.Is(verificationErr.Err, tc.expectedError) {
 				t.Errorf("VerifyTask got: %v, want: %v", err, tc.expectedError)
 			}
+			if int(verificationErr.resultType) != verificationResultError {
```

Comment:
Does this need to be an int? Can we compare the type values directly?

Reply:
oh no, fixed